src/commands/quote.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+export default class Quote {
+    constructor() {
+        this.INFO = {
+            command: 'quote',
+            description: 'get random quote',
+            standarize: true
+        }
+
+        this.VARIABLE = {
+            unknownError: 'Whoops... something error happened!',
+            url: 'http://quotes.stormconsultancy.co.uk/random.json'
+        }
+    }
+
+    async handle() {
+        return await this.getQuote().then(data => {
+
+            return data
+        });
+    }
+
+    async getQuote() {
+        return axios
```

Comment:
```js
try{
    return await axios.get(this.VARIABLE.url)
catch(err){
    return err.message || JSON.stringify(err)
}
```

Reply:
I am sorry 😄
this code ok?
`try {
        return await axios.get(this.VARIABLE.url)
            .then(response => {

                return `_${response.data.quote}_ - **${response.data.author}**`
            })
      } catch (err) {
        return err.message || JSON.stringify(err)
      }`

Reply:
no need to use `then` if you already using `await`

Reply:
```js
try{
    const response = await axios.get(this.VARIABLE.url)

    return `_${response.data.quote}_ - **${response.data.author}**`
}catch(err){
    return err.message || JSON.stringify(err)
}
```

Reply:
ah i see. Im so sorry for my bad

Reply:
thank you. i learn alot